pkgs/by-name/ti/tinyrange/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  makeWrapper,
+  qemu,
+}:
+buildGoModule (finalAttrs: {
+  pname = "tinyrange";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "tinyrange";
+    repo = "tinyrange";
+    tag = "v${finalAttrs.version}";
+
+    hash = "sha256-JdcINhbSpBQOhdlqRluYjtg8rbsy5iPA8rWn3kuUhVo=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  vendorHash = "sha256-WC2+HeuvBEyJVZaR2Eo98iQL8Nk4Q7e4+4jXljOQABg=";
+
+  env.CGO_ENABLED = 0;
+
+  # Remove the experimental directory; it messes with the build
+  postPatch = ''
+    rm -rf experimental
+  '';
+
+  subPackages = [
+    "cmd/tinyrange_qemu"
+    ""
```

Comment:
Is this empty string intended?

Reply:
Yes, it otherwise doesn't build the "root" package